client/signup/steps/design-type-with-store/index.jsx

```diff
@@ -36,6 +39,48 @@ export default React.createClass( {
 		this._pressableStore = pressableStoreStep;
 	},
 
+	scrollUp() {
+		// Didn't use setInterval in order to fix delayed scroll
+		while ( window.pageYOffset > 0 ) {
+			window.scrollBy( 0, -10 );
+		}
+	},
+
+	handleStoreBackClick() {
+		this.setState( {
+			showStore: false
+		} );
+
+		this.scrollUp();
+	},
+
+	handleChoiceClick( event, type ) {
+		event.preventDefault();
+		event.stopPropagation();
+		this.handleNextStep( type );
+	},
+
+	handleNextStep( designType ) {
+		analytics.tracks.recordEvent( 'calypso_triforce_select_design', { category: designType } );
```

Comment:
we will want to move this into a Redux action and out of a direct API call from within the component.

```js
import { recordTracksEvent } from 'state/analytics/actions';

handleNextStep( designType ) {
	this.props.recordNextStep( designType );
}

const mapDispatchToProps = dispatch => ( {
	recordNextStep: designType => dispatch( recordTracksEvent( 'calypso_triforce_select_design', { category: designType } ) )
} );
```